docs/source/complex_numbers.rst

```diff
@@ -0,0 +1,188 @@
+.. _complex_numbers-doc:
+
+Complex Numbers
+===============
+
+Complex Numbers are numbers that can be expressed in the form :math:`a + bi`, where a and b are real numbers,
+and *i* is a solution of the equation :math:`x^2 = −1`. Complex numbers frequently occur in mathematics and
+engineering, especially in signal processing. The aim of introducing tensors of complex dtypes is to provide
```

Comment:
Note that this documentation is going to live beyond its initial introduction, so you should try to talk about the feature as a timeless thing, and not like a release note.

Reply:
that's a very good point! Thanks Ed. I updated the doc but let me know if there are any more elements about it that are problematic